core/src/main/resources/SpringEnvironmentContext.xml

```diff
@@ -33,12 +34,6 @@
 
 	<bean id="NoopSecurityFilterChain" class="org.frankframework.lifecycle.servlets.NoopSecurityFilterChain" />
 
-	<bean id="httpFirewall" class="org.springframework.security.web.firewall.StrictHttpFirewall">
-		<property name="allowSemicolon" value="true"/>
```

Comment:
Ik weet niet zeker of het elkaar beinvloed, maar in `FrankConsoleContext.xml` staat alleen `allowSemicolon` en niet de twee hieronder. Is dat nog een risico voor de firewall config?

Reply:
Die staan er wel in.

Reply:
Ah, ingeklapt 🤦 . Ik moet nog steeds wennen aan github